caddy/caddy_test.go

```diff
@@ -467,29 +469,116 @@ func TestWorkerMetrics(t *testing.T) {
 	# TYPE frankenphp_busy_threads gauge
 	frankenphp_busy_threads 2
 
-	# HELP frankenphp_testdata_index_php_busy_workers Number of busy PHP workers for this worker
```

Comment:
Maybe could you keep the old test as is to ensure that there is no breaking change?

Reply:
@withinboredom has concerns about this as well https://github.com/dunglas/frankenphp/pull/1376#discussion_r1957185300